<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonDisconnectDevice">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>404</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect Device</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditLogs">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>911</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Read_Register">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>364</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Read Register</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_SetRegisterAdress">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Clouse">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>20</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ConnectTCP">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>404</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect TCP</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_L2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_Freq">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Start">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>290</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Stop">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>986</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMODBUS">
    <property name="title">
     <string>MODBUS</string>
    </property>
   </widget>
   <addaction name="menuMODBUS"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
